test/integration_test/snapshot_test.go

```diff
@@ -166,6 +171,62 @@ func groupSnapshotScaleTest(t *testing.T) {
 	destroyAndWait(t, allContexts)
 }
 
+func groupSnapshotRestoreTest(t *testing.T) {
+	ctxsToDestroy := make([]*scheduler.Context, 0)
+	// Positive tests
+	ctxs := createGroupsnaps(t, []string{
+		"mysql-groupsnap-restore",
+	})
+
+	ctxsToDestroy = append(ctxsToDestroy, ctxs...)
+
+	for _, ctx := range ctxs {
+		verifyGroupSnapshot(t, ctx, groupSnapshotWaitTimeout)
+	}
+
+	for _, ctx := range ctxs {
+		err := schedulerDriver.WaitForRunning(ctx, defaultWaitTimeout, defaultWaitInterval)
+		require.NoError(t, err, "Error waiting for pod to get to running state")
+
+		snaps, err := schedulerDriver.GetSnapshots(ctx)
+		logrus.Infof("Number of Snapshot: %d", len(snaps))
+		for _, snap := range snaps {
+			err = createRestorePvcForSnap(snap.Name, snap.Namespace)
```

Comment:
You also need to verify that the PVC gets bound. We want to make sure it doesn't fail.

Reply:
Fixed.